### PR TITLE
fix(list): filter work item by state close

### PR DESCRIPTION
**Pull request by vikram-raj**
<strong>Note: If there are pending changes to the PR, prefix the PR title with "WIP" and add the label "DO NOT MERGE"</strong>

### Mandatory
 - [x] What does this PR do? 
Filter work item by state close
- [x] What issue/task does this PR references?
https://github.com/openshiftio/openshift.io/issues/3955#issuecomment-404144985
- [x] Are the tests Included?
Yes

### CODE REVIEW

**Comment by vikram-raj** [comment]
What we can do is disable ```Show completed``` checkbox only when ```state``` filter is active. and enable it once the user is clear the ```state``` filter. WDYT @rgarg1?

**Comment by rgarg1** [comment]
Sounds good @vikram-raj . The checkbox should be disabled again if the user applies another state filter.

**Comment by vikram-raj** [comment]
> The checkbox should be disabled again if the user applies another state filter.

Yes, I will do this.

File: src/tests/specs/workItemTableTest.spec.ts [hunk 1]
```diff
@@ -196,4 +196,21 @@ describe('Work Item datatable list: ', () => {
...
+  it('Should filter the work item by close state', async () => {
+    let newWorkItem = {
+      title: 'Should filter the work item by close state - xxx'
+    };
+    let filterLabel = 'state: closed';
+    await planner.createWorkItem(newWorkItem);
+    expect(await planner.workItemList.hasWorkItem(newWorkItem.title)).toBeTruthy();
+    await planner.workItemList.clickWorkItem(newWorkItem.title);
+    await planner.quickPreview.changeStateTo('closed');
+    await planner.quickPreview.notificationToast.untilHidden();
```
Comment (Raunak1203) [inline src/tests/specs/workItemTableTest.spec.ts]: @vikram-raj could you please remove await planner.quickPreview.notificationToast.untilHidden();
Now only error notifications will be shown.
await planner.quickPreview.notificationToast.untilHidden();

**Comment by vikram-raj** [comment]
> The checkbox should be disabled again if the user applies another state filter.

@rgarg1 this changes is done. Please review again.

@Raunak1203 fixed the test. please review again.

**Comment by rgarg1** [comment]
LGTM @vikram-raj

**Pull request merged**